www/shared/tests/utils/render-with-intl/render-with-intl.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { renderWithIntl } from './render-with-intl';
+
+it('should correctly render formatted message', () => {
+    const { container, getByText } = renderWithIntl(
+        <FormattedMessage id="foo" />,
+    );
+
+    expect(container.innerHTML).toMatchSnapshot();
```

Comment:
IMO this assertion is unnecessary. 

`expect(getByText('foo')).toBeInTheDocument();` is enough to get 100% coverage.